.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+out
```

Comment:
pls, add .iml to gitignor and delete this file from project (this file generate IntelJIdea, so for this repository it's redundant)